src/Browser/Json.php

```diff
@@ -33,6 +33,53 @@ public function assertMatches(string $expression, $expected): self
         return $this;
     }
 
+    /**
+     * @param string $selector JMESPath selector
+     */
+    public function assertHas(string $selector): self
+    {
+        try {
+            $this->search("length({$selector})");
+            Assert::pass();
+        } catch (\RuntimeException $e) {
+            Assert::fail("Selector \"{$selector}\" does not exists.");
+        }
+
+        return $this;
```

Comment:
This is a pattern I use quite a bit with zenstruck/assert. [`Assert::try()`](https://github.com/zenstruck/assert#try) could be used here but it doesn't create a pass if it doesn't fail... maybe it should?

Reply:
I was indeed looking for something like this. Is there any cons about create a pass in `Assert:try()`?

Reply:
Not really I don't think. I use it in a few other library's to get a value before making the "real assertion".

Only con would be a few extra "assertions" registered by PHPUnit.

Can you think of a different method name that would act just like `try` but also make an assertion if no exception is thrown?

If not, I'm fine with having try create a pass itself.

Reply:
> Can you think of a different method name that would act just like try but also make an assertion if no exception is thrown?

I was actually more in favor of adding a pass in `Assert::try()` because I'm not able to find a good name :joy: